Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,262 @@
+                        PanOSApplicationSubcategory,
+                        "PanOSApplicationTechnology",
+                        PanOSApplicationTechnology,
+                        "PanOSDestinationDeviceOS",
+                        PanOSDestinationDeviceOS,
+                        "PanOSDestinationDeviceOSFamily",
+                        PanOSDestinationDeviceOSFamily,
+                        "PanOSDestinationDeviceOSVersion",
+                        PanOSDestinationDeviceOSVersion,
+                        "PanOSHostID",
+                        PanOSHostID,
+                        "PanOSHTTPHeaders",
+                        PanOSHTTPHeaders,
+                        "PanOSInlineMLVerdict",
+                        PanOSInlineMLVerdict,
+                        "PanOSInboundInterfaceDetailsType",
+                        PanOSInboundInterfaceDetailsType,
+                        "PanOSOutboundInterfaceDetailsType",
+                        PanOSOutboundInterfaceDetailsType,
+                        "PanOSParentSessionID",
+                        PanOSParentSessionID,
+                        "PanOSContainerName",
+                        PanOSContainerName,
+                        "PanOSContainerNameSpace",
+                        PanOSContainerNameSpace,
+                        "PanOSHTTPRefererFQDN",
+                        PanOSHTTPRefererFQDN,
+                        "PanOSHTTPRefererPort",
+                        PanOSHTTPRefererPort,
+                        "PanOSHTTPRefererProtocol",
+                        PanOSHTTPRefererProtocol,
+                        "PanOSHTTPRefererURLPath",
+                        PanOSHTTPRefererURLPath,
+                        "PanOSRuleUUID",
+                        PanOSRuleUUID,
+                        "PanOSURLCategoryList",
+                        PanOSURLCategoryList,
+                        "PanOSURLDomain",
+                        PanOSURLDomain,
+                        "PanOSURLCounter",
+                        PanOSURLCounter,
+                        "PanOSUsers",
+                        PanOSUsers,
+                        "PanOSVendorSeverity",
+                        PanOSVendorSeverity,
+                        "PanOSX-Forwarded-For",
+                        ["PanOSX-Forwarded-For"],
+                        "PanOSX-Forwarded-ForIP",
+                        ["PanOSX-Forwarded-ForIP"],
+                        "PanOSLogSource",
+                        PanOSLogSource
+                    ),
+          HttpContentType = RequestContext
+      | project-rename
+          DvcIpAddr = Computer,
+          EventUid = _ItemId,
+          DstDvcId = PanOSDestinationUUID,
+          DstGeoCountry = PanOSDestinationLocation,
+          DstMacAddr = PanOSDestinationDeviceMac,
+          DstNatIpAddr = DestinationTranslatedAddress,
+          DstNatPortNumber = DestinationTranslatedPort,
+          DstPortNumber = DestinationPort,
```

Comment:
@niralishah-crest @jayeshprajapaticrest 
in the official doc I found dpt as DestinationPort.
Can you please validate?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/643f4905-d46b-4036-9877-fad1bd0c4bb2)